R/uvhydrograph-styles.R

```diff
@@ -4,45 +4,75 @@
 #' @importFrom grDevices rgb
 getUvStyles <- function() {
   styles=list(
+      #raw signal should be green
+      uncorr_UV_lbl="Uncorrected UV",
+      uncorr_UV_lines=list(col="green", lty=4),
+      uncorr_UV2_lines=list(col="green", lty=4),
```

Comment:
Yep, it was only there because it was a different color before, but now I have merged the colors and so the style and renderer file were updated to use the not 2 style.